packages/frontend/src/components/MkEmojiPicker.vue

```diff
@@ -144,6 +153,39 @@ const searchResultCustom = ref<Misskey.entities.CustomEmoji[]>([]);
 const searchResultUnicode = ref<UnicodeEmojiDef[]>([]);
 const tab = ref<'index' | 'custom' | 'unicode' | 'tags'>('index');
 
+const customEmojiFolderRoot: CustomEmojiFolderTree = { value: "", category: "", children: [] };
+
+function parseAndMergeCategories(input: string, root: CustomEmojiFolderTree): CustomEmojiFolderTree {
+	const parts = input.split('/');
+	let currentNode: CustomEmojiFolderTree = root;
+
+	for (let part of parts) {
+		if (!part) {
+			part = '';
+		}
```

Comment:
```suggestion
	for (const part of parts) {
		if (!part) continue;
```

`part`の値がfalsyの場合は`currentNode`に入れたほうがいいんじゃないでしょうか

Reply:
そこをそう変えちゃうと先頭が`/`始まりのカテゴリとかを設定されてしまうとその絵文字が出てこなくなってしまうんですよね。

Reply:
あーrootノードの絵文字は描画されないのですね・・・

Reply:
とはいえこの部分ちょっとキモい記述なのでどうにかできそうならしたいところですね

Reply:
まっちゃとーにゅさんが別所で書いてたこれでいいやんとなったので丸パクリします・・・
https://github.com/MisskeyIO/misskey/pull/196/commits/09a9ab98305e86daf98c52f0c5e6482863b29073